### ISSUE TITLE
Create (static) 2d histogram of wealth distribution over time

### ISSUE THREAD

**Issue by WheezePuppet** [WheezePuppet/specnet#28]
That animated one is cool and all, but for the manuscript we do need a static figure.

**Comment by venkatachalapathy** [comment]
#29 is relevant for this one too.

**Comment by Temititayo** [comment]
For the manuscript, a series of histograms can be used.

**Comment by divilian** [comment]
> For the manuscript, a series of histograms can be used.
That's true, and perhaps that ends up being what we go with. I think it'll depend on whether a short series of histograms ends up being enough to "tell the tale." In the meantime, I think we probably do want to create the 2d heatmap for analysis purposes anyway. Assigning this one to you, @Temititayo.
